### PR TITLE
Projects

### PR DESCRIPTION
**Pull request by devp4**

### CODE REVIEW

**Comment by skbt27i** [comment]
It looks mostly good, but we should make sure that the assigned role is never an officer, and you could also check if the role is "open" (/roles command will tell you which are, and /openrole <rolename> will open a role).

**Review by skbt27i — CHANGES_REQUESTED** [review]
check if role is "open"

**Comment by devp4** [comment]
Current bug: if two users use /projects and one person clicks on a button, it triggers button for both users

**Comment by skbt27i** [comment]
are you able to fix that?

**Review by skbt27i — APPROVED** [review]

**Review by skbt27i — APPROVED** [review]

File: src/projects.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                "inline": true
+            },
+            {
+                "name": "Roles",
+                "value": "bot-dev",
+                "inline": true
+            },
+            {
+                "name": "Resources",
+                "value": "[Intro to TypeScript](https://youtu.be/gp5H0Vw39yw)\n[Discord.js Documentation](https://discord.js.org/#/)\n[Discrod.js Guide](https://discordjs.guide/#before-you-begin)",
```
Comment (skbt27i) [inline src/projects.json:18]: small typo here

**Review by skbt27i — APPROVED** [review]

**Pull request merged**